modules/ebi-metagenomics/hhsuite/buildhhdb/main.nf

```diff
@@ -0,0 +1,61 @@
+process HHSUITE_BUILDHHDB {
+    tag "$meta.id"
+    label 'process_high'
+
+    container 'microbiome-informatics/hh-suite-db-builder:1.0.0'
+
+    input:
+    tuple val(meta), path(a3m)
+
+    output:
+    tuple val(meta), path("${meta.id}"), emit: hh_db
```

Comment:
I'm not personally a fan of the output directory just being the meta.id, it's hard to know what it is without context... maybe add a suffix that describes what kind of data is in there, so `${meta.id}_hhsuitedb` or whatever makes most sense

Reply:
Agreed, changed to ${meta.id}_hh_db